errai-demos/errai-security-demo/pom.xml

```diff
@@ -310,6 +311,14 @@
           <extraJvmArgs>-Xmx1G -XX:CompileThreshold=7000 -Derrai.jboss.home=${errai.jboss.home} -Derrai.dev.context=${errai.dev.context}</extraJvmArgs>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${maven.deploy.plugin.version}</version>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
```

Comment:
Hi @tiagobento, @psiroky,

I've taken a quick look at how we build this because it intrigued me. It turns out we're using PME to override the deploy plugin (http://release-engineering.github.io/pom-manipulation-ext/guide/plugin-manip.html#install-and-deploy-skip-flag-alignment). So in short, whatever you use here is fine. :)